scorecard.js

```diff
@@ -0,0 +1,69 @@
+
+class ScoreCard {
```

Comment:
The syntax and structure of the parts you translated to JS look really good to me. Also clear/concise code throughout.